qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/update/UpdateOfferInput.groovy

```diff
@@ -0,0 +1,27 @@
+package life.qbic.portal.portlet.offers.update
+
+import life.qbic.datamodel.dtos.business.Offer
+
+/**
+ * Input interface for the {@link UpdateOffer} use case
+ *
+ * @since: 1.0.0
+ */
+interface UpdateOfferInput {
+
+    /**
+     * Searches for and existing offer and creates a new version with the provided information.
+     * The output of the use case is then informed about the successful creation of a new offer version
+     * <br>
+     * //<a href=" https://github.com/qbicsoftware/offer-manager-2-portlet/issues/222">TODO this information needs to contain changes to the previous offer</a>
+     * <br>
+     * <b>Precondition</b>:    Offer with identifier exists
+     * <br>
+     * <b>Postcondition</b>:   Offer is updated and has new version
+
+     * @param offerContent
+     * @since 1.0.0
+     */
+    void updateExistingOffer(Offer offerContent)
+
+}
```

Comment:
```suggestion
}

```

Reply:
Is there a reason you suggest a newline at the end of the file?

Reply:
yes. A newline char was missing and this is a violation of the POSIX standard. Please see [the Github issue](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) as reference.